### PR TITLE
Integraded home filter for users you haven't seen yet

### PR DESCRIPTION
**Pull request by joordy**
Integrated filter for users that you haven't seen yet. Made use of a static ID cause sessions doesn't work yet. When sessions work I can shorten my query to get all. the users.

My linter makes the code bigger, sorry.

### CODE REVIEW

File: routes/route.js [hunk 1]
```diff
@@ -122,17 +116,42 @@ async function postProfile(req, res, next) {
...
+  // To get myself out of array
+  return user.id === idLoggedIn;
 }
 
 async function home(req, res, next) {
   // Jordy & Veerle
   // Routes function home, graps every user with 'seen: false' and shows them on page.
   try {
-    let allUsers = await usersCollection.find({ seen: false }).toArray();
+    let database = await usersCollection.find().toArray(); // this code can be removed at the point sessions works.
```
Comment (veerleprins) [inline routes/route.js:132]: Why can this code be removed when sessions are working? I don't think searching for the whole collection can be removed with a session, right? In addition, the rest of your function will no longer work if you delete it I guess...

File: routes/route.js [hunk 2]
```diff
@@ -122,17 +116,42 @@ async function postProfile(req, res, next) {
...
     // code
   } catch (err) {
-      next(err);
-  }  
+    next(err);
+  }
+}
+
+function showMe(user) {
+  // To get myself out of array
```
Comment (veerleprins) [inline routes/route.js:124]: It's not yourself right, but the user that is logged in I guess?

File: routes/route.js (hunk 1 above)
Comment (joordy) [reply routes/route.js:132]: Line 132 till 135 can be removed when the session works, cause we can use `$ne: req.session.user.id` for example, to get the current user ID.

File: routes/route.js (hunk 2 above)
Comment (joordy) [reply routes/route.js:124]: Exact! Will change that.

**Pull request merged**